sympy/sets/fancysets.py

```diff
@@ -236,10 +236,10 @@ def _contains(self, other):
         from sympy.solvers.solveset import solveset, linsolve
         L = self.lamda
         if self._is_multivariate():
-            solns = list(linsolve([expr - val for val, expr in zip(other, L.expr)],
-                         L.variables).args[0])
+            solns = linsolve([expr - val for val, expr in zip(other, L.expr)],
+                             L.variables).args
```

Comment:
@aktech I think using `args[0]` will land us in the same spot as before.
I was thinking of getting the set of tuples returned by `linsolve` to `solns`.
Since, an  `EmptySet` won't have any `args` so `S.false` is returned.

I think this works but I would really appreciate your input !